enhancements/network-observability.md

```diff
@@ -0,0 +1,367 @@
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+Network Observability introduces a new category in OpenShift that
+provides networking information for a single cluster.  It gives insight
+into what's on the network, when and what types of traffic and traffic
+flows are being made, and by whom.  It gathers data to help design, plan,
+and answer questions about the network and provides a visual representation
+to help understand, diagnose, and troubleshoot networking issues.
+
+## Motivation
+
+With Kubernetes, a layer of abstraction is added making it difficult for
+Red Hat and customers who managed their networks to be able to fully see
+what's happening on their network.  Monitoring provides metrics and
+alerts to potential problems.  Network observability will then help you
+analyze, investigate and diagnose those problems by looking at it from
+a control plane perspective instead of device by device.  In addition,
+it can assist in the areas of network planning, network policy validation,
+security, and others.
+
+### Goals
+
+- Help troubleshoot network-related issues at the control plane level
+- Show what is happening on the network in a single cluster
+- Gather insight and be able to answer questions about the network
+- Provide rich visualization at the network layer, irrespective of the
+underlying layer
+
+The goal of the first release is to lay the groundwork and foundation
+in place, while still being able to deliver some functionality, even
+if it is at a smaller scale.  The target is to have a Dev Preview to
+generate interest in network observability.
+
+### Non-Goals
+
+The focus is on network observability as opposed to other types of observability.
+
+While a key goal is troubleshooting, it is not strictly a troubleshooting
+tool.  Some of that are already addressed by
+[Red Hat Insights](https://cloud.redhat.com/blog/openshift-insights-for-openshift-cluster-manager)
+and specific tools like pcap, traceroute, and [Jaeger](https://www.jaegertracing.io/)
+for tracing.
+
+
+## Proposal
+
+Network Observability is an opt-in feature that needs to be enabled
+by a user with an admin or cluster-admin role.  This is done by installing
+the Network Observability Operator, which in turn, installs the dependency
+operators necessary for this feature.  The user can do this using the
+web console or the CLI.
+
+In the initial release, Open vSwitch (OVS) will be configured to export
+IPFIX data.  The data will be collected and combined with Kubernetes-related
+information (e.g. pod, services, namespaces) and then saved in internal or
+external storage.
+
+The web console will provide a NetFlow table showing traffic between
+pods.  In the future, more visualization and functionality will be
+provided including areas such as topology, network-related eBPF data,
```

Comment:
I would re-word "network-related eBPF data", as eBPF is a a mechanism we may use to collect data, but not a type of data